syntax = "proto3";
package echo;
option cc_generic_services = true;


message EchoRequest {
    string msg = 1;
}

message EchoResponse {
    string msg = 2;
}

service EchoService {
    rpc Echo(EchoRequest) returns (EchoResponse);
}